requirements.txt

```diff
@@ -6,3 +6,4 @@ setuptools>=18.0
 -e git+https://github.com/typesupply/extractor.git@ufo3#egg=extractor
 -e git+https://github.com/typesupply/featools.git#egg=featools
 -e git+https://github.com/googlei18n/ufo2ft.git#egg=ufo2ft
+py2app==0.9
```

Comment:
I'd rather we don't put this in the requirements because much like Pyinstaller it's not required to run the program. Both are well-packaged and easy to install.